tigerpath/majors_and_certificates/majors/CLA_2018.json

```
@@ -0,0 +1,373 @@
+{
+  "type": "Major",
+  "name": "Classics",
+  "code": "CLA",
+  "degree": "AB",
+  "year": 2018,
+  "urls": [
+    "https://ua.princeton.edu/academic-units/department-classics",
+    "https://classics.princeton.edu/programs/undergraduate/tracks"
+  ],
+  "contacts": [
+    {
+      "type": "Chair",
+      "name": "Michael A. Flower",
+      "email": "mflower@princeton.edu"
+    },
+    {
+      "type": "Director of Undergraduate Studies",
+      "name": "Joshua Billings",
+      "email": "jhbillings@princeton.edu"
+    }
+  ],
+  "req_list": [
+    {
+      "name": "Tracks",
+      "max_counted": 1,
+      "min_needed": 1,
+      "explanation": null,
+      "req_list": [
+        {
+          "name": "Classics",
+          "max_counted": 1,
+          "min_needed": "ALL",
+          "explanation": null,

```

Again, no reason for it to be empty
```suggestion
          "explanation": "Classics uses knowledge of Greek or Latin (or both languages) as a gateway to the study of the literature, history, and culture of ancient Greece and Rome.\n\nStudents are expected to pass a sight translation examination from Greek or Latin. This examination may be taken, by arrangement, at the end of any term in the junior or senior year; it will be graded pass/fail.\n\nStudents who are considering further work in the field, either in graduate school or in independent study, should take both Latin and Greek to the 300 level, continuing with both languages in each term of the junior and senior years. Such students are also strongly advised to take at least one course in Greek history and one in Roman history in their underclass years.",
```